.github/workflows/build-test-deploy.yml

```diff
@@ -395,14 +373,5 @@ jobs:
         run: helm upgrade "${{ github.event.repository.name }}" ./deploy
           --install --timeout 60s
           --namespace "${{ github.event.repository.name }}"
-          --set image.tag="${{ github.sha }}"
```

Comment:
I'd recommend leaving it is. There is no `github.sha` variable defined in the Helm Chart.

Reply:
Ok, the other variables are also defined in the github actions context no? Things like secrets etc.